web/menu.jspf

```diff
@@ -219,7 +219,8 @@ org.opensolaris.opengrok.web.Util"
     <input tabindex="9" class="submit btn" type="submit" value="Search"/>
     <input tabindex="10" class="submit btn" onclick="javascript: clearSearchFrom();"
            type="button" value="Clear"/>
-    <a href="help.jsp" target="_blank" class="help btn" title="Help" tabindex="11" />Help</a>
+    <input tabindex="11" class="submit btn" onclick="window.open('help.jsp', '_blank');"
```

Comment:
shouldn't it be enough to just change the class ?

Reply:
No, the styles that would need to be added to achieve the same effect:
```css
font-weight: bold;
padding-top: 2px;
padding-bottom: 3px;
color: black; /* or something like that */
```

![screen shot 2017-10-20 at 21 01 50](https://user-images.githubusercontent.com/12401160/31837745-382a2e28-b5da-11e7-8d02-d9e45ced81aa.png)

As you can see on the screenshot, the padding was taken form `input` and for specific input type. I do not know if this style will be consistent across all browsers or if it won't change in the future. That's why I chose this implementation. (I might be wrong, I am not very skilled in frontend stuff)

Reply:
@tarzanek no that's not enought. I fought with this several times already

Reply:
looks like the consensus was reached, merging.